imports/ui/components/SearchBar.jsx

```diff
@@ -0,0 +1,294 @@
+import React from 'react';
+import clsx from 'clsx';
+import Select from 'react-select';
+import { emphasize, makeStyles, useTheme } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import NoSsr from '@material-ui/core/NoSsr';
+import TextField from '@material-ui/core/TextField';
+import Paper from '@material-ui/core/Paper';
+import Chip from '@material-ui/core/Chip';
+import MenuItem from '@material-ui/core/MenuItem';
+import CancelIcon from '@material-ui/icons/Cancel';
+import PropTypes from 'prop-types';
+
+const suggestions = [
+  { label: 'Apples' },
+  { label: 'Bananas' },
+  { label: 'Oranges' }
+].map(suggestion => ({
+  value: suggestion.label,
+  label: suggestion.label,
+}));
+
+const useStyles = makeStyles(theme => ({
```

Comment:
Let's move styling to the bottom since it's not the first thing we need to know what this component does, what do you think

Reply:
Moved styling to the bottom